dump/plugins/copp.py

```diff
@@ -207,6 +211,34 @@ def __get_asic_queue_obj(self, queue_sai_obj):
         if not ret["error"] and len(ret["keys"]) > 0:
             self.ret_temp["ASIC_DB"]["keys"].append(ret["keys"][0]) 
     
+    def __get_asic_hostif_entry_obj(self, sai_trap_key):
+        # Not adding tp tables_not_found because of the type of reason specified for policer obj
+        if not sai_trap_key:
+            return 
+        asic_obj_ptrn = "ASIC_STATE:.*:oid:0x\w{1,14}"
```

Comment:
I don't see the variable asic_obj_ptrn  being used. Please remove it if not used

Reply:
Dome